pinot-common/src/main/java/com/linkedin/pinot/common/utils/CommonConstants.java

```diff
@@ -246,7 +246,7 @@ public ServerType getServerType() {
     public static final long DEFAULT_QUERY_EXECUTOR_TIMEOUT_MS = 15_000L;
     public static final String DEFAULT_REQUEST_HANDLER_FACTORY_CLASS =
         "com.linkedin.pinot.server.request.SimpleRequestHandlerFactory";
-    public static final String DEFAULT_SEGMENT_LOAD_MAX_RETRY_COUNT = "5";
+    public static final String DEFAULT_SEGMENT_LOAD_MAX_RETRY_COUNT = String.valueOf(SegmentFetcher.RETRY_DEFAULT);
```

Comment:
Don't use the value from SegmentFetcher, it's separate config. Recommend change the type to int